### ISSUE TITLE
get_beta() module

### ISSUE THREAD

**Issue by SeaRoth** [lukaszbanasiak/yahoo-finance#100]
Is this the appropriate place to request a feature?

![image](https://cloud.githubusercontent.com/assets/3461713/22250867/748435d2-e1fd-11e6-93e4-016c3326cbfa.png)

**Comment by lukaszbanasiak** [comment]
Yes, pleas describe little bit more :) I'm guessing that you like to have `get_beta()` method that will returns data from screenshot?

**Comment by SeaRoth** [comment]
That is correct!  A feature that would return this data from the screenshot would be most helpful! =)

**Reasons:**
1. My beta-calculating algorithm isn't always precise
2. I'm trying to make my stock market app a better version of itself because of its inconsistent beta values

**Comment by SeaRoth** [comment]
I'm going to take a look at the source code - maybe I can add the feature myself and upload it for contribution

**Comment by aahmd** [comment]
I'd like to help as well but am having trouble finding the source code?

**Comment by lukaszbanasiak** [comment]
@SeaRoth you are welcome :)

@aahmd here https://github.com/lukaszbanasiak/yahoo-finance/blob/master/yahoo_finance/__init__.py

**Comment by aahmd** [comment]
@lukaszbanasiak awesome; thank you!

I figured the __init__ was blank (lol)
